packages/cerebral-provider-firebase/src/deleteUser.js

```diff
@@ -1,9 +1,13 @@
+import {noop as noReturnValue} from './helpers'
 import firebase from 'firebase'
+import {FirebaseProviderError} from './errors'
 
 export default function deleteUser (password) {
-  return new Promise((resolve, reject) => {
-    const user = firebase.auth().currentUser
+  const user = firebase.auth().currentUser
 
-    return user.delete().then(() => resolve(), (error) => reject({error}))
-  })
+  return user.delete()
+    .then(noReturnValue)
+    .catch((error) => {
+      throw new FirebaseProviderError(error.message)
```

Comment:
Looking at the source of `FirebaseProviderError` class it expects error objet but you're passing `message` directly in several actions. Will that result in message being undefined?